src/main/java/org/apache/commons/compress/archivers/zip/ZipSplitReadOnlySeekableByteChannel.java

```diff
@@ -155,30 +158,40 @@ public static SeekableByteChannel forOrderedSeekableByteChannels(final SeekableB
      * the beginning of a split archive
      */
     public static SeekableByteChannel buildFromLastSplitSegment(final File lastSegmentFile) throws IOException {
-        final String extension = FileNameUtils.getExtension(lastSegmentFile.getCanonicalPath());
+        return buildFromLastSplitSegment(lastSegmentFile.toPath());
+    }
+
+    /**
+     * Concatenates zip split files from the last segment(the extension SHOULD be .zip)
```

Comment:
Missing space between segment and parenthesis, and would SHOULD be replaced by MUST? It repeats in other javadocs further down in this file :point_down:

Reply:
hey, I changed in the places were I changed, but won't change in the places that were before, like Gary said to keep modifications only on the scope of the task.